### PR TITLE
docs: mention setUpLocationSync with $locationShim

### PR DESCRIPTION
**Pull request by jamie-pate**
#38415  PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

upgrade documentation for `$locationShim` doesn't mention `setUpLocationSync`

$locationShim doesn't seem to connect to the Location service at all
if you don't call `setUpLocationSync` from @angular/router/upgrade

Issue Number: N/A

## What is the new behavior?

Add a mention of `setUpLocationSync` which is required for hybrid routing to work

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by jamie-pate** [comment]
@googlebot I signed it!

File: aio/content/guide/upgrade.md [hunk 1]
```diff
@@ -952,10 +952,17 @@ For usage of the `$location` service as a provider in AngularJS, you need to dow
...
 import { downgradeInjectable } from '@angular/upgrade/static';
 
 angular.module('myHybridApp', [...])
   .factory('$location', downgradeInjectable($locationShim));
+
+...
+// if running in hybrid routing mode (optional):
+// call setUpLocationSync after bootstrap or routing won't be synced.
+this.upgrade.bootstrap(...);
+setUpLocationSync(this.upgrade);
```
Comment (gkalpak) [inline aio/content/guide/upgrade.md:965]: This feels a little out of place in this code snippet. It is not clear where this code should be placed and what `this.upgrade` is.

Maybe you could add a new code snippet below and have a small paragraph explaining where/how to call it.

**Review by gkalpak — CHANGES_REQUESTED** [review]
Thx, @jamie-pate! I think this is a useful addition, but we should expand a bit/have a more clear explanation and instructions.

**Comment by petebacondarwin** [comment]
@jamie-pate - are you able to address the comments from @gkalpak above so that we can merge this?

**Comment by petebacondarwin** [comment]
@gkalpak - I think we might close this PR since we have not heard back from @jamie-pate ??

**Comment by jamie-pate** [comment]
Sorry for the lack of response, server had a very busy release schedule, but Angular is finally part of fortios 7.0.0!

I'm not sure I can properly document this, but feel it should be documented, since it's still very confusing. Until an owner of the upgrade system has time to add proper docs for this feature this dead pr will have to serve as a clue to others who are trying to upgrade.

**Pull request closed without merging**